extensions/mssql/src/contracts.ts

```diff
@@ -36,19 +36,84 @@ export interface RequestSecurityTokenParams {
 	authority: string;
 	provider: string;
 	resource: string;
+	accountId: string;
 	scope: string;
 }
 
 export interface RequestSecurityTokenResponse {
 	accountKey: string;
 	token: string;
+	expiresOn: Number;
```

Comment:
You should revert these changes right? This isn't changing anything about the security token request/response